net/tcp/tcp_recvwindow.c

```diff
@@ -163,7 +165,29 @@ uint32_t tcp_get_recvwindow(FAR struct net_driver_s *dev,
       tailroom = 0;
     }
 
+  /* Get the number of buffers that the NIC needs to reserved */
+
+  niob_reserved = 0;
+  iob = dev->d_iob;
+
+  while (iob != NULL)
+    {
+      niob_reserved++;
+      iob = iob->io_flink;
+    }
+
+  niob_reserved = ((NETDEV_PKTSIZE(dev) + (CONFIG_IOB_BUFSIZE - 1)) /
```

Comment:
why need adjust niob_reserved

Reply:
The number of reserved iobs may already be in dev->d_iob, so the used part needs to be deleted